google-cloud-pubsub/OVERVIEW.md

```diff
@@ -306,36 +338,36 @@ require "google/cloud/pubsub"
 
 pubsub = Google::Cloud::PubSub.new
 
-sub = pubsub.subscription "my-topic-sub"
-subscriber = sub.listen do |received_message|
+subscriber = pubsub.subscriber "my-topic-sub"
+listener = subscriber.listen do |received_message|
   puts received_message.message.data
 
   # Mark for redelivery
   received_message.reject!
 end
 
 # Start background threads that will call the block passed to listen.
-subscriber.start
+listener.start
 
 # Shut down the subscriber when ready to stop receiving messages.
-subscriber.stop!
+listener.stop!
 ```
 
 Multiple messages can be delayed or made available for immediate redelivery:
-(See {Google::Cloud::PubSub::Subscription#modify_ack_deadline
-Subscription#modify_ack_deadline})
+(See {Google::Cloud::PubSub::Subscriber#modify_ack_deadline
+Subscriber#modify_ack_deadline})
 
 ```ruby
 require "google/cloud/pubsub"
 
 pubsub = Google::Cloud::PubSub.new
 
-sub = pubsub.subscription "my-topic-sub"
-received_messages = sub.pull immediate: false
-sub.modify_ack_deadline 120, received_messages
+subscriber = pubsub.subscriber "my-topic-sub"
+received_messages = subscriber.pull immediate: false
+subscriber.modify_ack_deadline 120, received_messages
 ```
 
-## Using Ordering Keys
+#### Using Ordering Keys
```

Comment:
I would make this an H3 (three hashes) so it encompasses the next few sections related to ordering.